src/aiida_quantumespresso/parsers/xspectra.py

```diff
@@ -159,22 +181,16 @@ def parse_stdout_xspectra(filecontent, codename=None, message_map=None):
 
     logs = get_logging_container()
     parsed_data = {}
+    # We need to assign this before the loop over each line due to a bug spotted in the code
+    # when `only_plot=True`. Since the epsilon vector is reported before the value of
+    # `xonly_plot`, we need a starting value before the loop begins.
+    # parsed_data['plot_only'] = False
```

Comment:
The code seems commented out? Is the comment actually correct? Or can this be removed?